README.md

```diff
@@ -698,3 +698,47 @@ module.settings.changeEdition = function (options, callback) {
   return callback(error, response)
 }
 ```
+
+#### getMappingMetadata
+getMappingMetadata function is used to add mapping validation for connectors to ensure certain business critical mappings are not removed/edited by a business user. This function is called on page refresh/load to get the mapping metadata for connector stack to make mappings non-editable or required (non-deletable) or both for a front end user.
+
+```js
+/*
+ * options object contains the following properties:
+ *     
+ *  bearerToken - a one-time bearer token which can be used to invoke selected integrator.io API routes.
+ *  
+ *  _integrationId - _id of the integration.
+ *
+ *  version - this field contains the version of the integration installed.
+ *
+ */
+
+module.settings.getMappingMetadata = function (options, callback) {
+  /*
+   *  settings function code
+   */
+
+  /*
+   *  err - return Error object but not handled in IO UI.
```

Comment:
@mujtaba2012 @ankursachdeva11 This is not very clear. Can we change it to match the description of other errs? Apart from it the rest of the PR looks fine.

Reply:
@ankush-garg, Updated error description. please review.